mastermindview/src/store/modules/socket.js

```diff
@@ -131,9 +149,13 @@ const mutations = {
     },
     CHANGE_PLAYER_ID(state, playerId) {
         state.socket.currentPlayerId = playerId
+    },
+    GET_CODE(state,playerId)
+    {
+        state.socket.currentPlayerId=playerId;
     }
```

Comment:
Dit kan weg, is niet nodig en wordt ook niet gebruikt dat ik zo snel kan zien

Reply:
do it

Reply:
doe wat?